### PR TITLE
[Filecoin] Update block explorer

### PR DESCRIPTION
**Pull request by hewigovens**
Changed to filfox.info

### CODE REVIEW

File: android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Thu May 14 14:21:48 CST 2020
+#Tue Dec 01 16:43:05 CST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
Comment (vikmeup) [inline android/gradle/wrapper/gradle-wrapper.properties:6]: Is it intended change?

**Review by vikmeup — APPROVED** [review]

File: android/gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (hewigovens) [reply android/gradle/wrapper/gradle-wrapper.properties:6]: I updated android studio and plugin

File: android/build.gradle [hunk 2]
```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.1'
+        classpath 'com.android.tools.build:gradle:4.1.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```
Comment (madcake) [inline android/build.gradle:11]: Could you update libraries too?

**Review by madcake — APPROVED** [review]

**Review by catenocrypt — APPROVED** [review]

File: android/build.gradle (hunk 2 above)
Comment (hewigovens) [reply android/build.gradle:11]: done

**Pull request merged**